SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -34,154 +34,74 @@ Worker_ComponentUpdate InterestFactory::CreateInterestUpdate()
 	return CreateInterest().CreateInterestUpdate();
 }
 
+// TODO - timgibson - change API so that we can avoid attaching empty Interest components.
 Interest InterestFactory::CreateInterest()
 {
 	if (!GetDefault<USpatialGDKSettings>()->bUsingQBI)
 	{
 		return Interest{};
 	}
 
-	if (Actor->GetNetConnection() != nullptr)
+	if (Actor->IsA(APlayerController::StaticClass()) && Actor->GetNetConnection() != nullptr)
 	{
-		return CreatePlayerOwnedActorInterest();
+		return CreateClientWorkerInterest();
 	}
-	else
-	{
-		return CreateActorInterest();
-	}
-}
-
-Interest InterestFactory::CreateActorInterest()
-{
-	Interest NewInterest;
-
-	QueryConstraint DefinedConstraints = CreateDefinedConstraints();
 
-	if (!DefinedConstraints.IsValid())
-	{
-		return NewInterest;
-	}
-
-	Query NewQuery;
-	NewQuery.Constraint = DefinedConstraints;
-	// TODO: Make result type handle components certain workers shouldn't see
-	// e.g. Handover, OwnerOnly, etc.
-	NewQuery.FullSnapshotResult = true;
-
-	ComponentInterest NewComponentInterest;
-	NewComponentInterest.Queries.Add(NewQuery);
-
-	// Server Interest
-	NewInterest.ComponentInterestMap.Add(SpatialConstants::POSITION_COMPONENT_ID, NewComponentInterest);
-
-	return NewInterest;
+	return Interest{};
```

Comment:
Isn't this going to break any multi worker capability we have at the moment since a server won't have a checkout radius anymore?